packages/sales-api-service/src/services/recurring-payments.service.js

```diff
@@ -153,6 +155,13 @@ export const cancelRecurringPayment = async id => {
   const recurringPayment = await findById(RecurringPayment, id)
   if (recurringPayment) {
     console.log('RecurringPayment for cancellation: ', recurringPayment)
+    const data = recurringPayment
+    data.entity.cancelledDate = new Date().toISOString()
+    data.entity.cancelledReason = await getGlobalOptionSetValue(RecurringPayment.definition.mappings.cancelledReason.ref, 'Payment failure')
+    const updatedRecurringPayment = Object.assign(new RecurringPayment(), data)
+    console.log(updatedRecurringPayment.entity)
+    const result = persist([updatedRecurringPayment.entity])
+    console.log(result)
```

Comment:
I'm assuming console logs are only for when working on this ticket and will be removed in next PR?

Reply:
This PR is still in progress though – it's https://github.com/DEFRA/rod-licensing/pull/2162 that needs reviewing